unassigned_pr/unassigned_pr_scraper.py

```diff
@@ -0,0 +1,66 @@
+import json
+import requests
+
+from github import Github
+from django.http import HttpResponse
+from gci.config import get_api_key
+
+from community.git import get_org_name
+
+
+def run(issues, GH_TOKEN, reponame):
+    issues_number_list = []
+    for j in issues:
+        issue_no = j.number
+        events = j.get_events()
+        data = []
+        for i in events:
+            if i.commit_id is not None:
+                data.append(str(i.commit_id))
+        extracted_data = data[-1:]
+        if len(extracted_data) == 1:
+            commit_id = extracted_data[0]
+            url = 'https://api.github.com/search/issues?q='+commit_id + \
+                '+type:pr+repo:'+reponame  # Ignore PycodestyleBear (E501)
+            r = requests.get(url, headers={'Authentication':
+                                           'token '+GH_TOKEN,
+                                           'Accept':
+                                           'application/vnd.github.symmetra-preview+json'})  # Ignore PycodestyleBear (E501)  and LineLengthBear
+
+            # check commit is a opened as PR or not.
+            if r.json()['total_count'] > 0:
```

Comment:
Can you show me the json you are getting with `r.json()`?

Reply:
The output is in this format
```
/home/vaibhav/test/backup.py
````

pastebin link https://pastebin.com/5DLEZBBQ

Reply:
Getting same error 
`CommandError: Failed to render view: 'total_count'`

Here is the netlify link of my fork. https://app.netlify.com/sites/raivaibhav-fork/deploys/5acc9ebfdd28ef5456638e1f